source/standards/source-code/use-github.html.md.erb

```diff
@@ -52,12 +52,16 @@ Consider protecting the `.github/workflows` folder by using [a CODEOWNERS file](
 Consider creating a Workflow Template in the [alphagov workflow folder](https://github.com/alphagov/.github/tree/main/workflow-templates) if you need to share a similar workflow between many repositories.
 
 [Create your own local actions](https://docs.github.com/en/actions/creating-actions/about-actions) wherever possible. If using GitHub-owned actions, [pin to a specific version](https://docs.github.com/en/actions/learn-github-actions/finding-and-customizing-actions#using-release-management-for-your-custom-actions).
+Pinned versions should include the semver version in a comment next to the SHA, helping humans understand which versions we are pinned to.
+Where possible, allow automated dependency management tools to scan these versions and suggest updates.
```

Comment:
`versions` -> `version comments`

Worth adding a link to the comment formats that are supported too